setup.py

```diff
@@ -21,7 +21,7 @@
     long_description=long_description,
     long_description_content_type='text/markdown',
     license='CC0',
-    version='4.1.1',
+    version='4.1.2',
```

Comment:
If we're dropping Wagtail 1.13 support, we should bump the version to 4.2.
```suggestion
    version='4.2.0',
```

Reply:
Done.